easybuild/easyconfigs/h/HEALPix/HEALPix-3.50-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'HEALPix'
+version = '3.50'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+
+# basic_gcc, generic_gcc, optimized_gcc targets available
+gcc_target = 'optimized_gcc'
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['Healpix_3.50_2018Dec10.tar.gz']
+patches = ['HEALPix_noXtest.patch']
+checksums = [
+    'ec9378888ef8365f9a83fa82e3ef3b4e411ed6a63aca33b74a6917c05334bf4f',  # Healpix_3.50_2018Dec10.tar.gz
+    'ec82c8b2beb80937c83d2e545e26ade08b3647ab5a3401c8703e6523b0782c08',  # HEALPix_noXtest.patch
+]
+
+dependencies = [('CFITSIO', '3.45')]
+
+builddependencies = [('binutils', '2.30')]
```

Comment:
You can drop this line if you use `GCC` rather than `GCCcore` as toolchain (since `binutils` is a part of `GCC`)